frontend/src/components/maps/leaflet/mapUtils.tsx

```diff
@@ -113,7 +114,10 @@ export const getMarkerIcon = (feature: ICluster) => {
     ].includes(projectStatus)
   ) {
     return erpIcon;
-  } else if (projectNumber !== undefined) {
+  } else if (
```

Comment:
checking to see if the project number is populated as a way of choosing the purple pin doesn't match up with how our filters work.

Reply:
We don't generally return the project status of a property with the geoJSON endpoints.  Neither do we want to moving forward, as it will be another hit on performance.  We may need to rethink how the map displays coloured pins.

Reply:
ok... but is checking the classification like this ok? the projectStatus stuff predates my change.